stories/sparkline.stories.js

```diff
@@ -29,7 +29,7 @@ const id_multi = "multiVis";
 const lineConfig = { interpolate: "monotone" };
 
 export default {
-  title: "SparkLine",
+  title: "Components/Chart/SparkLine",
```

Comment:
For maintainability purposes, we could assign variables for each topic/subtopic.
These values should be aligned with the `.storybook/preview.js` ones.

Reply:
yes, agree, and it would be easier to change/to maintain.